lms/static/sass/discussion/views/_inline.scss

```diff
@@ -0,0 +1,63 @@
+// forums - inline discussion styling
+// ====================
+
+.discussion.inline-discussion {
+    padding-top: $baseline;
+
+    .inline-threads {
+        border: 1px solid $forum-color-border;
+        border-radius: $forum-border-radius;
+    }
+
+    .inline-thread {
+        border: 1px solid $forum-color-border;
+        border-radius: $forum-border-radius;
+
+        .forum-nav-bar {
+            background-color: $forum-color-navigation-bar;
```

Comment:
NIT: I would suggest adding padding: ($baseline/4); here 

Also, @chris-mike - At one point we talked about making the background of this bar not blue. Are you ok with the blue or should we downplay this area to a dark gray / non-primary color?